scalding-core/src/main/scala/com/twitter/scalding/estimation/memory/MemoryHistoryEstimator.scala

```diff
@@ -0,0 +1,99 @@
+package com.twitter.scalding.estimation.memory
+
+import com.twitter.scalding.estimation.{ FlowStepHistory, FlowStrategyInfo, HistoryEstimator, MemoryEstimatorConfig, Task }
+import org.apache.hadoop.mapred.JobConf
+import org.slf4j.LoggerFactory
+
+// Tuple(MapMemory in MB, ReduceMemory in MB), or None to keep the default.
+case class MemoryEstimate(mapMemory: Option[Long], reduceMemory: Option[Long])
```

Comment:
Suggestion: Add `Mb` to the field names to reduce chance of misusing these params